### ISSUE TITLE
[BUG] Broken Tabs closable

### ISSUE THREAD

**Issue by mgawino** [bokeh/bokeh#11683]
Bokeh 2.4.0, Python 3.8.10, Ubuntu 20.04.3 LTS, Firefox 92.0

Closing a tab produces an error (executed with bokeh server). Related issue [here](https://discourse.bokeh.org/t/close-a-tab-error-raise/8079
).
Minimal example:
```
from bokeh.io import curdoc
from bokeh.models import Panel, Tabs, Div

panel1 = Panel(child=Div(), title='Panel1')
panel2 = Panel(child=Div(), title='Panel2', closable=True)

tabs = Tabs(tabs=[panel1, panel2])

curdoc().add_root(tabs)
```
Traceback

> 
> 2021-09-26 18:24:44,716 Starting Bokeh server version 2.4.0 (running on Tornado 6.1)
> 2021-09-26 18:24:44,717 User authentication hooks NOT provided (default user enabled)
> 2021-09-26 18:24:44,718 Bokeh app running at: http://localhost:5006/app2
> 2021-09-26 18:24:44,718 Starting Bokeh server with process id: 152874
> 2021-09-26 18:24:45,222 WebSocket connection opened
> 2021-09-26 18:24:45,223 ServerConnection created
> 2021-09-26 18:24:59,349 error handling message
>  message: Message 'PATCH-DOC' content: {'events': [{'kind': 'ModelChanged', 'model': {'id': '1005'}, 'attr': 'tabs', 'new': [{'id': '1002'}]}], 'references': []} 
>  error: DeserializationError("Instance(Panel) failed to deserialize reference to {'id': '1002'}")
> Traceback (most recent call last):
>   File "/home/mateusz/test/venv/lib/python3.8/site-packages/bokeh/server/protocol_handler.py", line 97, in handle
>     work = await handler(message, connection)
>   File "/home/mateusz/test/venv/lib/python3.8/site-packages/bokeh/server/session.py", line 93, in _needs_document_lock_wrapper
>     result = func(self, *args, **kwargs)
>   File "/home/mateusz/test/venv/lib/python3.8/site-packages/bokeh/server/session.py", line 287, in _handle_patch
>     message.apply_to_document(self.document, self)
>   File "/home/mateusz/test/venv/lib/python3.8/site-packages/bokeh/protocol/messages/patch_doc.py", line 115, in apply_to_document
>     invoke_with_curdoc(doc, lambda: doc.apply_json_patch(self.content, setter))
>   File "/home/mateusz/test/venv/lib/python3.8/site-packages/bokeh/document/callbacks.py", line 408, in invoke_with_curdoc
>     return f()
>   File "/home/mateusz/test/venv/lib/python3.8/site-packages/bokeh/protocol/messages/patch_doc.py", line 115, in <lambda>
>     invoke_with_curdoc(doc, lambda: doc.apply_json_patch(self.content, setter))
>   File "/home/mateusz/test/venv/lib/python3.8/site-packages/bokeh/document/document.py", line 391, in apply_json_patch
>     DocumentPatchedEvent.handle_json(self, event_json, references, setter)
>   File "/home/mateusz/test/venv/lib/python3.8/site-packages/bokeh/document/events.py", line 259, in handle_json
>     handler(doc, event_json, references, setter)
>   File "/home/mateusz/test/venv/lib/python3.8/site-packages/bokeh/document/events.py", line 467, in _handle_json
>     patched_obj.set_from_json(attr, value, models=references, setter=setter)
>   File "/home/mateusz/test/venv/lib/python3.8/site-packages/bokeh/core/has_props.py", line 384, in set_from_json
>     descriptor.set_from_json(self, json, models=models, setter=setter)
>   File "/home/mateusz/test/venv/lib/python3.8/site-packages/bokeh/core/property/descriptors.py", line 376, in set_from_json
>     value = self.property.prepare_value(obj, self.name, self.property.from_json(json, models=models))
>   File "/home/mateusz/test/venv/lib/python3.8/site-packages/bokeh/core/property/container.py", line 83, in from_json
>     return self._new_instance([ self.item_type.from_json(item, models=models) for item in json ])
>   File "/home/mateusz/test/venv/lib/python3.8/site-packages/bokeh/core/property/container.py", line 83, in <listcomp>
>     return self._new_instance([ self.item_type.from_json(item, models=models) for item in json ])
>   File "/home/mateusz/test/venv/lib/python3.8/site-packages/bokeh/core/property/instance.py", line 112, in from_json
>     raise DeserializationError(f"{self} failed to deserialize reference to {json}")
> bokeh.core.property.bases.DeserializationError: Instance(Panel) failed to deserialize reference to {'id': '1002'}

**Comment by mattpap** [comment]
Most likely a duplicate of issue #11422, which is fixed by PR #11469.

**Comment by bryevdv** [comment]
I confirmed that things are broken on `branch-3.0` but fixed the branch for PR #11469 so I will close this as a dupe.
